nativescript-angular/directives/dialogs.ts

```diff
@@ -17,15 +18,15 @@ import { AppHostView } from "../app-host-view";
 import { DetachedLoader } from "../common/detached-loader";
 import { PageFactory, PAGE_FACTORY } from "../platform-providers";
 import { once } from "../common/utils";
-import { topmost, Frame } from "tns-core-modules/ui/frame";
+import { topmost, Frame, ShowModalOptions } from "tns-core-modules/ui/frame";
```

Comment:
I think `ShowModalOptions`  should be imported from "tns-core-modules/ui/core/view" above to avoid confusion.

Reply:
Agreed.